.expeditor/create_release.sh

```diff
@@ -27,7 +27,7 @@ function download_artifacts {
       zip -j "${util_name}_${os}_${arch}.zip" "${file}"
       files+=("${util_name}_${os}_${arch}.zip")
     else
-      tar -czf "${file}_${os}_${arch}.tar.gz" -C "go-binaries/${os}/${arch}" .
+      tar -czf "${file}_${os}_${arch}.tar.gz" --exclude "${file}_${os}_${arch}.tar.gz" -C "go-binaries/${os}/${arch}" .
```

Comment:
since we know the name of the file we want to tar up why not explicitly specify it instead of using `.`? That way we should be able to get rid of the `--exclude`.

```suggestion
      tar -czf "${file}_${os}_${arch}.tar.gz" -C "go-binaries/${os}/${arch}" "${file}"
```

Reply:
makes sense